src/olympia/reviewers/templates/reviewers/base.html

```diff
@@ -28,8 +28,8 @@ <h1 class="site-title amo">
         {% endif %}
       </nav>
     {% endif %}
-</div>
   {% endblock %}
+</div>
```

Comment:
That `</div>` closes `<div class="reviewer-menu">`, so it needs to be after the `{% block pillnav %}{% endblock %}` if we want to be able to override that block. That block had never been overridden before, that's why we didn't notice it was in the wrong place.